### PR TITLE
[OSDOCS-9995]: Add AWS CLI requirement to "Getting started with the ROSA CLI" topic in ROSA docs

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-9995
Link to docs preview:

https://73350--ocpdocs-pr.netlify.app/openshift-rosa/latest/cli_reference/rosa_cli/rosa-get-started-cli#rosa-setting-up-cli_rosa-getting-started-cli
QE review:
- [x] QE has approved this change.

Additional information:

**Reviewers: Scope of ticket includes adding AWS CLI install as part of setting up ROSA CLI. Extent of change was copy/paste from one section to another, and a change to one word per PM suggestion. Thanks.**

### CODE REVIEW

File: modules/rosa-setting-up-cli.adoc [hunk 1]
```diff
@@ -12,6 +12,28 @@ Use the following steps to install and configure the ROSA CLI (`rosa`) on your i
...
 .Procedure
 
+. Install and configure the latest AWS CLI (`aws`).
+.. Follow the link:https://aws.amazon.com/cli/[AWS Command Line Interface] documentation to install and configure the AWS CLI for your operating system.
++
+Specify your `aws_access_key_id`, `aws_secret_access_key`, and `region` in the `.aws/credentials` file. See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS Configuration basics] in the AWS documentation.
++
+[NOTE]
+====
+You can alternatively use the `AWS_DEFAULT_REGION` environment variable to set the default AWS region.
```
Comment (arendej) [inline modules/rosa-setting-up-cli.adoc:22]: swap 'alternatively' with 'optionally'

**Comment by mletalie** [comment]
Hello @xueli181114  When you get a chance can you please provide QE review for this PR https://github.com/openshift/openshift-docs/pull/73350? Thanks!

**Review by arendej — APPROVED** [review]
lgtm

**Comment by yuwang-RH** [comment]
lgtm

**Pull request merged**